viewing_party/party.py

```diff
@@ -1,23 +1,176 @@
 # ------------- WAVE 1 --------------------
 
+# from tkinter.tix import InputOnly
+
+
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:

```

Comment:
Excellent use of the guard clause pattern! Early return here means no unnecessary code execution!